README.md

```diff
@@ -1,65 +1,86 @@
+x = st.slider("Select a value")
+st.write(x, "squared is", x * x)
 ```
 
-<img src="https://raw.githubusercontent.com/streamlit/docs/main/public/images/simple_example.png"/>
+<img src="https://user-images.githubusercontent.com/7164864/214900507-1c89fc6b-e196-4f5c-890f-7e0be62d5d9d.png" width=300 alt="Little example"></img>
 
-## A bigger example
+#### Give me more!
 
-Streamlit's simple and focused API lets you build incredibly rich and powerful tools.  [This demo project](https://github.com/streamlit/demo-self-driving) lets you browse the entire [Udacity self-driving-car dataset](https://github.com/udacity/self-driving-car) and run inference in real-time using the [YOLO object detection net](https://pjreddie.com/darknet/yolo).
+Streamlit comes in with [a ton of additional handy elements](https://docs.streamlit.io/library/api-reference) to spice your data apps and delight your viewers. Some examples:
 
-![Final App Animation](https://raw.githubusercontent.com/streamlit/docs/main/public/images/complex_app_example.gif)
+<table border="0">
+   <tr>
+     <td><strong>Input widgets</strong><br><br><img src="https://docs.streamlit.io/images/api/date_input.jpg" height="150"></td>
+     <td><strong>Celebrations<br><br><img src="https://docs.streamlit.io/images/api/balloons.jpg" height="150"></td>
+     <td><strong>Multi-page apps<br><br><img src="https://docs.streamlit.io/images/mpa-add-pages.png" height="150"></td>
+     <td><strong>Charts<br><br><img src="https://docs.streamlit.io/images/api/line_chart.jpg" height="150"></td>
+     <td><strong>Layout<br><br><img src="https://docs.streamlit.io/images/api/tabs.jpg" height="150"></td>
+     <td><strong>Media<br><br><img src="https://docs.streamlit.io/images/api/image.jpg" height="150"></td>
+  </tr>
+<table>
 
-The complete demo is implemented in less than 300 lines of Python. In fact, the app contains [only 23 Streamlit calls](https://github.com/streamlit/demo-self-driving/blob/master/streamlit_app.py) which illustrates all the major building blocks of Streamlit. You can try it right now at [share.streamlit.io/streamlit/demo-self-driving](https://share.streamlit.io/streamlit/demo-self-driving).
+Our vibrant creators community also extends Streamlit capabilities using [Streamlit Components](https://streamlit.io/components) 🧩
 
-## The Streamlit GitHub badge
+## Get inspired 
+ 
+There's so much you can build with Streamlit!
+- 🧬  [Science & technology apps](https://streamlit.io/gallery?category=science-technology)
+- 💬  [NLP & language apps](https://streamlit.io/gallery?category=nlp-language)
+- 👀  [Computer vision & images](https://streamlit.io/gallery?category=computer-vision-images)
+- 🏦  [Finance & business](https://streamlit.io/gallery?category=finance-business)
+- 🗺  [Geography & society](https://streamlit.io/gallery?category=geography-society)
+- ...
 
-Streamlit's GitHub badge helps others find and play with your Streamlit app.
+**Discover more in [our gallery](https://streamlit.io/gallery)** 🎈
```

Comment:
We could think about having a nice image that shows a bunch of apps here. But can also add this later on, doesn't need to block v1. And maybe it's even too much distraction.